jsoc/gsod/projects.md

```diff
@@ -66,19 +66,23 @@ Just to name a few. One project would be to create a unified scientific machine
   This would also greatly benefit the adoption of Flux in the larger ML ecosystem, which we feel is currently held back due to not having enough of these simple patterns documented in an approachable form. We want to see an increase in the number of contributors to the various packages too since that would help us improve our stack better. Flux also utilizes simple to understand and performant code, made possible by Julia, and through this, we also want to bring awareness to how our ecosystem has matured, and increase its adoption in research and industry.
 
 
-## Turing (Probabilistic Machine Learning)
+## Turing (Probabilistic machine learning)
```

Comment:
```suggestion
## Turing (probabilistic machine learning)
```

It should probably be all lowercase or all uppercase initial letters, it's a bit weird in-between here.